xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/ConversionMatrix.cpp

```diff
@@ -473,7 +473,6 @@ void CConvertMatrix::GenMat()
   CTranslate trans(0, -0.5, -0.5);
   *m_pMat *= trans;
 
-  m_limitedSrc = 8;
```

Comment:
@FernetMenta I think this shouldn't be here

Reply:
oops. I put it there for debugging and forgot to remove. thanks!